temporalio/temporalio.gemspec

```diff
@@ -12,13 +12,11 @@ Gem::Specification.new do |spec|
   spec.homepage = 'https://github.com/temporalio/sdk-ruby'
   spec.license = 'MIT'
   spec.required_ruby_version = '>= 3.1.0'
-  spec.required_rubygems_version = '>= 3.3.11'
 
   spec.metadata['homepage_uri'] = spec.homepage
   spec.metadata['source_code_uri'] = 'https://github.com/temporalio/sdk-ruby'
 
-  spec.files = Dir['lib/**/*.rb', 'ext/**/*.*', 'Cargo.lock', 'Cargo.toml', 'Gemfile', 'Rakefile',
-                   'temporalio.gemspec', 'LICENSE', 'README.md']
+  spec.files = Dir['lib/**/*.rb', 'LICENSE', 'README.md', 'Cargo.*']
```

Comment:
What directive here ensures that the compiled bridge libraries gets included in the final gem?

Reply:
Why include Cargo.*?